src/library_wasmfs.js

```diff
@@ -5,6 +5,18 @@
  */
 
 mergeInto(LibraryManager.library, {
+  $MEMFS: {
+    createBackend: (opts) => (_wasmfs_create_memory_backend())
+  },
+  $ICASE: {
```

Comment:
Should we give all the these globals names with `FS` in them.. how about `ICASEFS`, and `FETCHFS`?   

Also, so we need to even other with creating these globals for non-legacy filesystems?  Maybe can skip it?

Reply:
I agree with `ICASEFS`, etc. 

I think having the globals for the non-legacy filesystems like `NODEFS` made the implementation a little easier, since I could match the legacy JS API of `FS.mount(NODEFS)` (passing an object), then give the object a `createBackend()` method to make the correct backend to pass to `wasmfs_mount`. Would it be better to not use globals for each FS and try a different approach?

Reply:
I guess I was thinking the JS API only needs to support enough for compatibility with the legacy FS.  

Maybe for new filesystems there doesn't need to be an API for mounting from JS at all?

Reply:
I see, would this mean that the new JS API won't support using the new filesystems from JS at all? i.e. the new backends can only be used in C/C++?